BeeSwift/Gallery/GalleryViewController.swift

```diff
@@ -147,7 +147,7 @@ class GalleryViewController: UIViewController, UICollectionViewDelegateFlowLayou
             make.top.equalTo(self.searchBar.snp.bottom)
             make.left.equalTo(self.view.safeAreaLayoutGuide.snp.leftMargin)
             make.right.equalTo(self.view.safeAreaLayoutGuide.snp.rightMargin)
-            make.bottom.equalTo(0)
+            make.bottom.equalTo(self.collectionView!.keyboardLayoutGuide.snp.top)
```

Comment:
👀 [keyboardLayoutGuide](https://developer.apple.com/documentation/uikit/uiview/3752221-keyboardlayoutguide)

looks great! 👏 
This one line using this iOS 15 feature sure beats out the manual 27 line approach in #503.

(_bothers me that we have another use of collectionView! - we could just mark it as such at Line 23 and not have to do it anywhere else instead of doing it everywhere else, though obviously this is not directly related to adjusting the view's bottom to handle the keyboard_)